src/main/webapp/frame.jsp

```diff
@@ -51,7 +51,7 @@
           <div class="region-sidebar-left col-sm-2 col-xs-0 hidden-xs no-margin" ng-if="$storage.showSidebar && APP_FLAGS.showSidebar">
               <side-bar-menu></side-bar-menu>
           </div>
-          <div ng-if="(!($storage.showSidebar)) && APP_FLAGS.showSidebar" class="show-sidebar" ng-click="$storage.showSidebar = true">
+          <div ng-if="(!($storage.showSidebar)) && APP_FLAGS.showSidebar" class="show-sidebar" ng-click="$storage.showSidebar = true" onClick="pushGaEvent('Sidebar','Maximize Sidebar', 'Maximize Sidebar')">
```

Comment:
This won't work as `onClick` wouldn't have the context of Angular. You will need to do a `ng-click` and add a function to the controller.  Possibly a "toggleSidebar" function, which just fires the proper GA event and switches the `$storage.showSidebar`.